.pre-commit-config.yaml

```diff
@@ -1,14 +1,14 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: 21.12b0
+    rev: 22.3.0
```

Comment:
To ensure we keep these things up to date (and consistent across repos) -  what's the consensus on adding autoupdate in our workflows as here: https://pre-commit.com/#:~:text=pre%2Dcommit%20autoupdate%20%5Boptions%5D